src/org/lice/parse2/Token.java

```diff
@@ -3,25 +3,51 @@
 import org.lice.model.MetaData;
 
 public class Token {
-	public enum TokenKind {
-		NumericLiteral,
+	public enum TokenType {
+		BinNumber,
+		OctNumber,
+		DecNumber,
+		HexNumber,
+		LongInteger,
+		BigInteger,
+		FloatNumber,
+		DoubleNumber,
 		StringLiteral,
 		Identifier,
 		LispKwd,
 		EOI
 	}
```

Comment:
Missing `BigDecimal`.

Reply:
Recommended to use `BigInt` and `BigDec`, to avoid naming conflict.

Reply:
Fixed in https://github.com/lice-lang/lice-tiny/pull/1/commits/b67e48911be370300900bfbeabb22863a0dc3681 . It was I misunderstood the requirement at first.

Reply:
Good!